content/courses/Ada_For_The_C_Embedded_Developer/chapters/04_Embedded.rst

```diff
@@ -782,7 +782,85 @@ Previously, we've seen that we can use
 layout for a record type. As mentioned before, this is useful when interfacing
 with hardware, drivers, or communication protocols. In this section, we'll
 extend this concept for two specific use-cases: register overlays and data
-streams.
+streams. Before we discuss those use-cases, though, we'll first explain the
+:ada:`Size` aspect and the :ada:`Size` attribute.
+
+.. _Size_Aspect_Attribute:
+
+:ada:`Size` aspect and attribute
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ada:`Size` aspect indicates the minimum number of bits required to
+represent an object, and it can be used to confirm expectations. When applied
```

Comment:
I suggest removing "and it can be used to confirm expectations" because that placement early in the explanation gives that particular usage primacy, whereas it is one of three usages. Also, the text mentions confirmation soon after (where I think it belongs).